.travis.yml

```diff
@@ -63,8 +63,7 @@ jobs:
       before_install:
         - sudo apt install -y scons
       install:
-        # TODO for Ubuntu Focal: faad=0 ffmpeg=1
-        - scons -j "$(nproc)" faad=1 ffmpeg=0 localecompare=1 mad=1 modplug=1 wv=1
+        - scons -j "$(nproc)" faad=0 ffmpeg=1 localecompare=1 mad=1 modplug=1 wv=1
```

Comment:
faad=0 could be omitted. Sorry, I forgot to update both comment and settings in the CI PR.